app/src/main/kotlin/com/saltedge/authenticator/features/settings/about/AboutListFragment.kt

```diff
@@ -87,21 +90,15 @@ class AboutListFragment : BaseFragment(), OnItemClickListener {
         }
     }
 
-    private fun injectDependencies() {
-        activity?.let {
-            DaggerAboutListComponent.builder().aboutListModule(AboutListModule(it)).build().inject(this)
-        }
-    }
-
     private fun setupViewModel() {
         viewModel = ViewModelProviders
             .of(this, viewModelFactory)
             .get(AboutViewModel::class.java)
 
-        viewModel.licenseItemClickEvent.observe(this, Observer<ViewModelEvent<Unit>> {
+        viewModel.licenseItemClickEvent.observe(this, Observer {
             it?.let { activity?.addFragment(LicensesFragment()) }
         })
-        viewModel.termsOfServiceItemClickEvent.observe(this, Observer<ViewModelEvent<Bundle>> {
+        viewModel.termsOfServiceItemClickEvent.observe(this, Observer {
```

Comment:
I want to see type `<ViewModelEvent<Bundle>> ` in fragment, not only in `viewModel`